CHANGELOG.md

```diff
@@ -14,6 +15,16 @@ All changes to the project will be documented in this file.
 * Improved performance of find implementations (PR: [#1935](https://github.com/OmniSharp/omnisharp-roslyn/pull/1935))
 * Add support for new quick info endpoint when working with Cake (PR: [#1945](https://github.com/OmniSharp/omnisharp-roslyn/pull/1945))
 * Add support for new completion endpoints when working with Cake ([#1939](https://github.com/OmniSharp/omnisharp-roslyn/issues/1939), PR: [#1944](https://github.com/OmniSharp/omnisharp-roslyn/pull/1944))
+* When an analyzer fails to load, log an error (PR: [#1972](https://github.com/OmniSharp/omnisharp-roslyn/pull/1972))
+* Bump Roslyn to 3.8.0-4.20503.2 (PR: [#1974](https://github.com/OmniSharp/omnisharp-roslyn/pull/1974))
+* Updated build tool version for NET 5 RC2 (PR: [#1971](https://github.com/OmniSharp/omnisharp-roslyn/pull/1971))
+* Added support for 'extract base class' (PR: [#1969](https://github.com/OmniSharp/omnisharp-roslyn/pull/1969))
+* Moved most test asset projects to .NET Core 3.1 (PR: [#1968](https://github.com/OmniSharp/omnisharp-roslyn/pull/1968))
+* Upgraded to SDK 3.1.402 (PR: [#1966](https://github.com/OmniSharp/omnisharp-roslyn/pull/1966))
+* OmniSharp.Path can only be set in user settings (PR: [#1946](https://github.com/OmniSharp/omnisharp-roslyn/pull/1946))
+* Adds support for /completion and /completion/resolve endpoints for Cake. (PR: [#1944](https://github.com/OmniSharp/omnisharp-roslyn/pull/1944))
```

Comment:
this is  duplicate

Reply:
I didn't see what this was duplicate of but hopefully with the update the problem is solved.